setup.py

```diff
@@ -36,6 +38,7 @@ def find_version(*file_paths):
     url = "https://github.com/bram85/topydo",
     install_requires = [
         'six >= 1.9.0',
+        'arrow >= 0.6.0',
```

Comment:
It's in PyPi now.

You can leave it at 0.6 then, but please make sure nothing funny happens with the latest version (that people are likely going to install when upgrading topydo).